www/templates/html/SiteMaster/Core/Auditor/QATestController.tpl.php

```diff
@@ -0,0 +1,25 @@
+<p>
+    URL To Test: <?php echo $context->url ?>
+</p>
+<?php if (!$context->current_user): ?>
+    <p>We were not able to find a previous scan for this page.  You must log in to continue.</p>
+<?php elseif (!$context->site): ?>
+    <p>
+        We don't know about this site.  Please add it and we will scan it!
+        <a href="<?php \SiteMaster\Core\Config::get('URL') ?>/sites/add/" class="button wdn-button">Register the site now</a>
```

Comment:
Is that the case this conditional is trying to catch? I would assume site.com is not going to refer to this URL. 

The best guess would be the first found directory path and the system was not configured for the domain, you could ignore the link altogether, as evening sending them to the add site page would just add another click to their expected failure.